### PR TITLE
syncing javadoc files

### PR DESCRIPTION
**Pull request by carimura**

### CODE REVIEW

**Review by ammbra — APPROVED** [review]
All good here.

**Comment by danthe1st** [comment]
This seems to break the JavaFX Javadocs for me (locally via Docker). For example, when clicking the `Stage` link in `JavaFX Application Basic Structure By Example` (`/learn/javafx/structure/`), it forwards me to https://download.java.net/java/GA/javafx22/@@RELEASE_UUID@@/docs/api/javafx.graphics/javafx/stage/Stage.html which is an invalid link.

**Review by danthe1st — CHANGES_REQUESTED** [review]
This seems to break the JavaFX Javadocs for me (locally via Docker). For example, when clicking the `Stage` link in `JavaFX Application Basic Structure By Example` (`/learn/javafx/structure/`), it forwards me to https://download.java.net/java/GA/javafx22/@@RELEASE_UUID@@/docs/api/javafx.graphics/javafx/stage/Stage.html which is an invalid link.

File: app/data/javafxdoc.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "current_release":                                  "19",
+  "current_release":                                "21.0.1",
+  "release_uuid":                                     "e5ab43c6aed54893b0840c1f2dcfca4d",
 
-  "javafx-documentation":                             "https://openjfx.io/javadoc/@@CURRENT_RELEASE@@/",
-  "javafxdoc_root":                                   "https://openjfx.io/javadoc/@@CURRENT_RELEASE@@/",
+  "javafxdoc_root":                                   "https://download.java.net/java/GA/javafx@@CURRENT_RELEASE@@/@@RELEASE_UUID@@/docs/api/",
```
Comment (danthe1st) [inline app/data/javafxdoc.json:5]: This seems to break the JavaFX Javadocs for me (locally via Docker). For example, when clicking the `Stage` link in `JavaFX Application Basic Structure By Example` (`/learn/javafx/structure/`), it forwards me to https://download.java.net/java/GA/javafx22/@@RELEASE_UUID@@/docs/api/javafx.graphics/javafx/stage/Stage.html which is an invalid link.

**Review by danthe1st — CHANGES_REQUESTED** [review]

**Comment by danthe1st** [comment]
It seems to work now except for the `RotationTransition` link in `JavaFX Properties` (`/learn/javafx/properties/`) and I think that was missing before as well.

**Review by JosePaumard — APPROVED** [review]

**Pull request merged**